modules/mobile/app/services/mobile/v0/lighthouse_health/service.rb

```diff
@@ -32,6 +32,9 @@ def get_immunizations
         #
         def get_location(id)
           response = perform(:get, "Location/#{id}", nil, headers)
+          # i don't see any evidence this is happening. not sure why. this code appears to be in active use
+          # i actually find the very confusing because the code that calls this is actively logging
+          # regardless, we probably don't need this
           Rails.logger.info('Mobile Lighthouse Service, Location response', response:)
```

Comment:
Category: data discovery
Action: I couldn't find this in the logs despite the fact that it has to be happening. I suspect it's being omitted because it's trying to expose the entire response and that's causing issues. Changing code to only expose the response body. Once I confirm it worked, delete this plus comments in controller.